generators/app/templates/src/StarterKit/Startup/LoggingEngineExtensions.cs

```diff
@@ -32,19 +34,19 @@ public static IServiceCollection AddLoggingEngine(this IServiceCollection servic
       return services;
     }
 
-    public static ILoggerFactory AddLoggingEngine(this ILoggerFactory loggerFactory, IApplicationBuilder app, IApplicationLifetime appLifetime, IConfiguration config)
+    public static ILoggerFactory AddLoggingEngine(this ILoggerFactory loggerFactory, IApplicationBuilder app, IHostApplicationLifetime appLifetime, IConfiguration config)
     {
-      var enrichers = app.ApplicationServices.GetServices<ILogEventEnricher>().ToArray();
+      var enrich = app.ApplicationServices.GetServices<ILogEventEnricher>().ToArray();
 
       var systemLogSection = config.GetSection("SystemLog");
       var applicationLogSection = config.GetSection("ApplicationLog");
 
       var appLogger = typeof(ApplicationLogger).FullName;
 
       Log.Logger = new LoggerConfiguration()
-                      .Enrich.With(enrichers)
-                      .WriteTo.Logger(l => l.ReadFrom.ConfigurationSection(systemLogSection).Filter.ByExcluding(Matching.FromSource(appLogger)))
-                      .WriteTo.Logger(l => l.ReadFrom.ConfigurationSection(applicationLogSection).Filter.ByIncludingOnly(Matching.FromSource(appLogger)))
+                      .Enrich.With(enrich)
+                      .WriteTo.Logger(l => l.ReadFrom.Configuration(systemLogSection).Filter.ByExcluding(Matching.FromSource(appLogger)))
+                      .WriteTo.Logger(l => l.ReadFrom.Configuration(applicationLogSection).Filter.ByIncludingOnly(Matching.FromSource(appLogger)))
```

Comment:
Alles m.b.t. ApplicationLogger mag weg. Er worden geen aparte indexen meer aangemaakt voor System en Application.

Reply:
In de toekomst komen er nog aparte indexen maar zullen logs waarschijnlijk opgesplits worden op basis van tags die meegegeven worden. Lijkt me inderdaad niet meer nodig dan.